tests/import_export/import.py

```diff
@@ -24,31 +26,95 @@
+                 user=None):
     with open(os.path.join(file_dir, file_name), "r") as f:
-        import_file(SimpleUploadedFile(file_name,
-                                       f.read(),
-                                       content_type))
+        import_file(
+            SimpleUploadedFile(file_name, f.read(), content_type),
+            user=user)
 
 
 @pytest.mark.django_db
-def test_import_success(en_tutorial_po_no_file):
+def test_import_success(en_tutorial_po_no_file, admin):
     assert en_tutorial_po_no_file.state == NEW
-    _import_file(IMPORT_SUCCESS)
+    _import_file(IMPORT_SUCCESS, user=admin)
     store = Store.objects.get(pk=en_tutorial_po_no_file.pk)
     assert store.state == PARSED
 
 
 @pytest.mark.django_db
-def test_import_failure(file_import_failure, en_tutorial_po):
+def test_import_failure(file_import_failure, en_tutorial_po, member):
     filename, exception = file_import_failure
     with pytest.raises(exception):
-        _import_file(filename)
+        _import_file(filename, user=member)
 
 
 @pytest.mark.django_db
-def test_import_unsupported(en_tutorial_ts, ts_directory):
+def test_import_unsupported(en_tutorial_ts, ts_directory, member):
     with pytest.raises(UnsupportedFiletypeError):
         _import_file(IMPORT_UNSUPP_FILE,
                      file_dir=os.path.join(ts_directory, "tutorial/en"),
-                     content_type="text/vnd.trolltech.linguist")
+                     content_type="text/vnd.trolltech.linguist",
+                     user=member)
+
+
+@pytest.mark.django_db
+def test_import_new_file(import_tps, site_users):
+    tp = import_tps
+    user = site_users["user"]
+    store_pootle_path = tp.pootle_path + "import_new_file.po"
+    filestore = create_store(store_pootle_path, "0",
+                             [("Unit Source", "Unit Target")])
+
+    # New store can't be created via current import command. This test will
+    # need to be adjusted if we allow to create new stores via import command.
+    from import_export.exceptions import FileImportError
+    with pytest.raises(FileImportError):
+        import_file(SimpleUploadedFile("import_new_file.po",
+                                       str(filestore),
+                                       "text/x-gettext-translation"), user)
+
+
+@pytest.mark.django_db
+def test_import_to_empty(import_tps, site_users):
+    from pytest_pootle.factories import StoreFactory
+
+    tp = import_tps
+    user = site_users["user"]
+    store = StoreFactory(translation_project=tp, name="import_to_empty.po")
+    filestore = create_store(store.pootle_path, "0",
+                             [("Unit Source", "Unit Target")])
+    import_file(SimpleUploadedFile(store.name,
+                                   str(filestore),
+                                   "text/x-gettext-translation"), user)
+
+    if (tp.is_template_project
+        or tp.project.checkstyle == 'terminology') and user.is_superuser:
```

Comment:
This should be checking if user is a manager (has `administrate` permission) not if it is a superuser.